cmd/kube-controller-manager/app/options/options.go

```diff
@@ -104,6 +104,7 @@ func NewCMServer() *CMServer {
 			ClusterSigningCertFile:   "/etc/kubernetes/ca/ca.pem",
 			ClusterSigningKeyFile:    "/etc/kubernetes/ca/ca.key",
 			ReconcilerSyncLoopPeriod: metav1.Duration{Duration: 5 * time.Second},
+			EnableTaintManager:       true,
```

Comment:
OK - I'm really confused now. Is this going to be alpha or beta? I.e. we plan on moving Taints to beta as a field, will 'NoExecute' effect be an alpha value for that field? How do we gate/mark this?